README.md

```diff
@@ -1,8 +1,27 @@
 # Deploy a Testnet
 
-## Software Setup \(Manual Installation\)
+## Setup
```

Comment:
Let's remove this section about cloud deployment. A lot of people think that you need to deploy it to a cloud server. Also, if you already have a cloud server you can just execute the below commands to install it there.

We could add a section on how to install go correctly though.